packages/shared/src/recording/getRecordings.ts

```diff
@@ -273,3 +326,13 @@ export function getRecordings(processGroupIdFilter?: string): LocalRecording[] {
       .sort((a, b) => b.date.getTime() - a.date.getTime())
   );
 }
+
+function logNotFoundWarning(recordingId: string, entries: LogEntry[], entry: LogEntry) {
+  logger.error(`Recording with ID "${recordingId}" not found`, {
```

Comment:
the recently added logs are in a format like this:
```suggestion
  logger.error(`RecordingLog:RecordingNotFound`, {
```